net-mail/vacation/vacation-1.2.7.0-r1.ebuild

```diff
@@ -17,25 +17,19 @@ RDEPEND="virtual/mta
 DEPEND="${RDEPEND}
 	!mail-mta/sendmail"
 
-src_unpack() {
-	unpack ${A}
-	cd "${S}"
-}
-
 src_prepare() {
-	eapply_user
+	default
 
 	sed -i -e "s:install -s -m:install -m:" Makefile || die "sed failed!"
```

Comment:
why not multiple regexs sed?

Reply:
Hmm, if I understand correctly you suggest to merge 2 sed calls into one?
Wouldn't do this, the execution time won't drop and the maintenance gets harder with 1 sed call

Reply:
Or maybe I misunderstood you. Do you got a code example?

Reply:
Of course it does, you don't have to open and close the same file twice. Less I/O, memory etc.

Besides it's easier to see what is being done to a single file, less chance of failures...